hawkbit-repository/hawkbit-repository-test/src/main/java/org/eclipse/hawkbit/repository/test/TestConfiguration.java

```diff
@@ -77,7 +79,12 @@
 @EnableAutoConfiguration
 @PropertySource("classpath:/hawkbit-test-defaults.properties")
 public class TestConfiguration implements AsyncConfigurer {
-
+    @Bean
+    public ServletRegistrationBean h2servletRegistration() {
```

Comment:
Not sure if the H2 console is required all the time for every integration test

Reply:
I'd prefer to enable the console only on demand during a local debug session

Reply:
removed...